templates/frontEndLayout.html

```diff
@@ -98,133 +108,133 @@
 <table>
```

Comment:
This entire table is about to get a major overhaul.  Rather than manually having to update each change 9 times, we can have the server loop through 3x3 times and write each <td> out for us.  This will make the code easier to maintain and avoid copy/paste errors where you forget to update one id in the middle of a random plot.

To do this, we're going to use jinja2 templating, which is built into Flask.  It'll let us put commands in the html file that the server will execute before sending the text down.  For instance, if your .html file looked like:
```
<table>
{% for i in range(3) %}
    <tr>
    {% for j in range(3) %}
        <td><div id="plot{{ i }}{{ j }}"></div></td>
    {% endfor %}
    </tr>
{% endfor %}
</table>
```
you'd get a very similar structure.  The `{% %}` and `{{ }}` will be executed and removed on the server side, so the browser will just see the finished html and not know the difference.